src/components/App/Search/Search.jsx

```diff
@@ -18,7 +18,7 @@ export default class Search extends Component {
     };
 
     listSongs() {
-        if (this.props.songs && this.props.songs.length > 0) {
+        if (this.props.songs && this.props.songs.length > 0 && this.props.query.length > 0) {
```

Comment:
If there's nothing typed in the input, don't show anything.

Reply:
seems sensible. there shouldn't be any songs without any query